finetune/comparison.py

```diff
@@ -1,9 +1,39 @@
 import numpy as np
+import tensorflow as tf
 
 from finetune.base import BaseModel
-from finetune.classifier import Classifier
+from finetune.classifier import Classifier, ClassificationPipeline
 from finetune.encoding import ArrayEncodedOutput
-import tensorflow as tf
+
+
+class ComparisonPipeline(ClassificationPipeline):
+
+    def _format_for_encoding(self, X):
```

Comment:
In the development branch the _format_for_encoding function is consistent for all `Model` classes sans sequence_labeling.  I think we can move this into the ClassificationPipeline and prevent having to redefine this in every subclass?